templates/zerver/emails/realm_deactivated.html

```diff
@@ -0,0 +1,26 @@
+{% extends "zerver/emails/email_base_default.html" %}
+
+{% block illustration %}
+<img src="{{ email_images_base_url }}/email_logo.png" alt=""/>
+{% endblock %}
+
+{% block content %}
+<p>
+    {% if no_acting_user %}
+        {% trans %}Your Zulip organization, {{ realm_name }}, was deactivated on {{ event_date }}.{% endtrans %}
+    {% else %}
+        {% if is_acting_owner %}
+            {% trans %}You have deactivated your Zulip organization, {{ realm_name }}, on {{ event_date }}.{% endtrans %}
+        {% else %}
+            {% trans %}Your Zulip organization, {{ realm_name }}, was deactivated by {{ deactivating_owner }} on {{ event_date }}.{% endtrans %}
+        {% endif %}
+    {% endif %}
+</p>
+<p>
+    {% if corporate_enabled %}
+        {% trans contact_support="https://zulip.com/help/contact-support" %}If you have any questions or concerns, please <a href="{{ contact_support }}">contact Zulip Support</a> as soon as possible.{% endtrans %}
```

Comment:
> The issue has the Zulip Cloud footer linking to the help center article Contact Support and I just wanted to confirm that's the correct and we don't want to link directly to the support email address directly in the email.

@alya thoughts on this detail?  `git grep help/contact-support templates/zerver/emails/` suggests we don't link to that page elsewhere in emails, and we probably do want a direct email reply in most cases ... so it could make sense to just rewrite both cases with "please reply to this email as soon as possible".

Reply:
Yeah, that sounds good as long as the from: email will be correct for just sending a reply.

Reply:
I updated this so that the link in the Zulip Cloud email is a "mailto" link to the support email address.

If we instead want to be able to reply to these emails, then we'd set the from address to `FromAddress.SUPPORT`?

Reply:
I think we can set `reply-to` of `FromAddress.SUPPORT` independently from `From`, if that's helpful.